doc/metrics.md

```diff
@@ -21,6 +22,211 @@ _Metrics API Reference: <https://open-telemetry.github.io/opentelemetry-js-api/c
   - [Exporting measurements to Prometheus](#exporting-measurements-to-prometheus)
   - [Exporting measurements to Opentelemetry Protocol](#exporting-measurements-to-opentelemetry-protocol)
 
+## Getting Started
+In thie page, you'll leaarn how to setup and metrics from an HTTP server with Fastify. If you're not using Fastify, that's fine -- this guide will also work with Express, etc.
```

Comment:
```suggestion
In this page, you'll learn how to setup OpenTelemetry JS to export metrics from an HTTP server with Fastify. If you're not using Fastify, that's fine -- this guide will also work with Express, etc.
```

Reply:
Thank you @dyladan . I have applied your suggestions and made sure some words are spelled in American English form